lookup.go

```diff
@@ -53,6 +59,12 @@ func loggableKey(k string) logging.LoggableMap {
 	}
 }
 
+func multihashLoggableKey(mh multihash.Multihash) logging.LoggableMap {
+	return logging.LoggableMap{
+		"multihash": base32.RawStdEncoding.EncodeToString(mh),
+	}
```

Comment:
This is uppercase, but when we encode CIDs using b32 we use lowercase. Should I lowercase this?